scripts/deploy.sh

```diff
@@ -24,12 +24,21 @@ export DEPLOYMENT_STAGE=$1
 # EOF
 # }
 
+function delete_all_job_definitions() {
+    jobNames=$(aws batch describe-job-definitions | jq -r '.jobDefinitions[].jobDefinitionName')
+    for jobName in $jobNames; do
```

Comment:
This function will delete all job definitions regardless of component or deployment stage. We should figure out how to delete only the definitions relevant to a specific deployment of the upload service using variable $DEPLOYMENT_STAGE

Reply:
Done... I think.

Reply:
Looks good